iroh-net/src/magic_endpoint.rs

```diff
@@ -1118,9 +1255,10 @@ mod tests {
         let _ep2_guard = CallOnDrop::new(move || {
             ep2_abort_handle.abort();
         });
-        async fn accept(ep: MagicEndpoint) -> (PublicKey, String, quinn::Connection) {
+        async fn accept(ep: MagicEndpoint) -> NodeId {
```

Comment:
that's surprising, the `Connection` gets dropped immediately here?

Reply:
Yeah, the original test did not rely on this connection at all, it just wants to assert on the NodeId.  So I simplified this to make that clear.  :shrug: